README.md

```diff
@@ -14,6 +14,9 @@ to be installed locally via pip.
 Also, the Innoactive Hub's images are distributed via Innoactive's private Docker registry so in order to access and deploy
 them, credentials to access the docker registry at `registry.docker.innoactive.de` are required.
 
+This role expects docker as well as pip docker to be installed.
```

Comment:
That's very short ...

Better something like:

> Since the Innoactive Hub consists of a number of services each of which is bundled as a docker container, this role requires the ansible host to have [docker](https://www.docker.com/) installed (API level >= 1.20) and ready to use. To ensure this, you can e.g. use the excellent ansible role [geerlingguy.docker](https://galaxy.ansible.com/geerlingguy/docker). 
>
> Additionally, ansible's [docker module](https://docs.ansible.com/ansible/latest/modules/docker_container_module.html) requires the `docker` pip module to be installed in order for Python to be able to communicate with the running docker service. Installing this package can e.g. either be done via the native ansible [pip module](https://docs.ansible.com/ansible/latest/modules/pip_module.html) or via a dedicated role like [geerlingguy.pip](https://galaxy.ansible.com/geerlingguy/pip).